lib/docker-sync/dependencies/package_manager.rb

```diff
@@ -0,0 +1,18 @@
+module DockerSync
+  module Dependencies
+    module PackageManager
+      class << self
+        extend Forwardable
+        def_delegators :package_manager, :available?, :ensure!, :install_package
+      end
+
+      def self.package_manager
+        return @package_manager if defined? @package_manager
+        return @package_manager = PackageManager::Brew if PackageManager::Brew.available?
```

Comment:
IIUC.. we want to make `brew` a hard requirement under osx.. so we need to update this to `return @package_manager = PackageManager::Brew if Environment.mac?`..

else, the next lines will continue be evaluated, and we will end up with `PackageManager::None`..